store/tikv/coprocessor.go

```diff
@@ -424,6 +433,10 @@ type copIteratorWorker struct {
 	replicaReadSeed uint32
 
 	sendRate *rateLimit
+
+	actionOnExceed *rateLimitAction
+
+	tasksStatus *copTaskStatusHandler
```

Comment:
Is it seem that we do not need to define copTaskStatusHandler?
We only need to add an attribute `maxRunningTaskID *atomic.Value` for `copIterator` and `copIteratorWorker`